### PR TITLE
rps3

### PR DESCRIPTION
**Pull request by Asteroid384**
initial

### CODE REVIEW

File: src/RPS/App.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+    <startup> 
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
```
Comment (mattkins99) [inline src/RPS/App.config:4]: Are you sure you want this 4.7.2?

Comment (Asteroid384) [reply src/RPS/App.config:4]: yes

**Review by mattkins99 — COMMENTED** [review]
LGTM

**Review by mattkins99 — APPROVED** [review]
LGTM

**Pull request merged**